src/main/scala/com/fulcrumgenomics/umi/ReviewConsensusVariants.scala

```diff
@@ -0,0 +1,139 @@
+import scala.collection.mutable
+import scala.collection.mutable.ListBuffer
+
+@clp(group=ClpGroups.Umi, description=
+  """
+    |Extracts data to make reviewing of variant calls from consensus reads easier. Creates
+    |a list of variant sites from the input VCF or IntervalList and then extracts all the
+    |consensus reads that do not contain a reference allele at the the variant sites, and
+    |all raw reads that contributed to those consensus reads.  This will include consensus
+    |reads that carry the alternate allele, a third allele, a no-call or a spanning
+    |deletion at the variant site.
+    |
+    |Both input BAMs must be coordinate sorted and indexed.
+    |
+    |A pair of output BAMs named {output}.consensus.bam and {output}.grouped.bam
+    |are created with the relevant reads from each input BAM.
+  """)
+class ReviewConsensusVariants
+( @arg(flag="i", doc="Input VCF or IntervalList of variant locations.") val input : FilePath,
+  @arg(flag="c", doc="BAM file of consensus reads used to call variants.")  val consensusBam : PathToBam,
+  @arg(flag="g", doc="BAM file of grouped raw reads used to build consensuses.") val groupedBam : PathToBam,
+  @arg(flag="r", doc="Reference fasta file.") val ref: PathToFasta,
+  @arg(flag="o", doc="Basename of output files to create.") val output : PathPrefix,
+  @arg(flag="t", doc="The SAM/BAM tag used to uniquely identify source molecules.") val tag: String = "MI"
+)extends FgBioTool {
+  Io.assertReadable(Seq(input, consensusBam, groupedBam, ref))
+  Io.assertCanWriteFile(output)
+  private val refFile = ReferenceSequenceFileFactory.getReferenceSequenceFile(ref)
+  private val dict = refFile.getSequenceDictionary
+
+  private[umi] case class LocAndRef(chrom: String, start: Int, refBase: Char) {
+    def toQueryInterval: QueryInterval = new QueryInterval(dict.getSequenceIndex(chrom), start, start)
+  }
+
+  override def execute(): Unit = {
+    def f(ext: String): Path = output.getParent.resolve(output.getFileName + ext)
+    val consensusIn = SamReaderFactory.make().open(consensusBam)
+    val groupedIn   = SamReaderFactory.make().open(groupedBam)
+    val factory = new SAMFileWriterFactory().setCreateIndex(true)
+    val consensusOut = factory.makeWriter(consensusIn.getFileHeader, true, f(".consensus.bam").toFile, ref.toFile)
+    val groupedOut   = factory.makeWriter(groupedIn.getFileHeader, true, f(".grouped.bam").toFile, ref.toFile)
+
+    // Pull out the consensus reads
+    val variants        = loadPositions(path=input, refFile=refFile)
+    val variantsByChrom = variants.groupBy(v => v.chrom)
+    val queries         = variants.map(_.toQueryInterval).toArray
+    val sources         = mutable.HashSet[String]()
+    consensusIn.query(queries, false).filter(r => nonReferenceAtAnyVariant(r, variantsByChrom)).foreach { rec =>
+      consensusOut.addAlignment(rec)
+      Option(rec.getStringAttribute(tag)).map(mi => mi.takeWhile(_ != '/')).foreach(sources.add)
```

Comment:
Add a doc in the usage that all characters after the first `/` will be ignored in the `MI` tag?